CONTRIBUTING.md

```diff
@@ -1,7 +1,28 @@
-# Contributing
+# Contributing to panther-analysis
 
-Please follow the [Code of Conduct](https://github.com/panther-labs/panther-analysis/blob/main/CODE_OF_CONDUCT.md)
-in all of your interactions with the project.
+Thank you for your interest in contributing to Panther's open-source ruleset!  We value and appreciate all contributions, whether they are bug reports, feature requests, or detection rule contributions.
+
+## What Makes a Good Detection?
+
+Please familiarize yourself with these helpful resources on writing high-quality Panther rules.  Especially excellent contributions will be considered for seasonal prizes!
+
+- Jack's blog on [The Anatomy of a High Quality SIEM Rule](https://jacknaglieri.substack.com/p/hq-siem-rules)
+- Panther's [Detection Documentation](https://docs.panther.com/detections)
+- panther-analysis [Style Guide](https://github.com/panther-labs/panther-analysis/blob/main/STYLE_GUIDE.md)
+
+## Testing Your Changes
+
+Before submitting your pull request, make sure to:
+
+- Write or update relevant unit tests.
+- Redact any sensitive information or PII from example logs.
+- Format, lint, and test your changes to ensure CI tests pass.
+
+```bash
```

Comment:
can you move this codeblock under the last bullet point above? adding: "to ensure CI tests pass, using the following commands:"